src/riak_core_coverage_plan.erl

```diff
@@ -759,6 +764,17 @@ covers(KeySpace, CoversKeys) ->
 data_bits(PartitionCount) ->
     160 - round(math:log(PartitionCount) / math:log(2)).
 
+%% @private
+
+%% Crimes against computerkind. Finds the next value of two greater
+%% than the requested value using string manipulation.
+next_power_of_two(X) ->
+    Next = 1 bsl length(hd(io_lib:format("~.2b", [X]))),
```

Comment:
log2 isn’t present in R16.

I could do some magic with log, but it would probably be slower than the current implementation.

-John

> On Dec 31, 2015, at 10:29 AM, Torben Hoffmann <notifications@github.com> wrote:
> 
> round(math:pow(2,round(math:log2(N-1) + 0.5)))